go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/alecthomas/kingpin/v2 v2.4.0
 	github.com/aws/aws-sdk-go v1.55.5
 	github.com/go-kit/kit v0.13.0
+	github.com/go-kit/log v0.2.1
```

Comment:
is `github.com/go-kit/log` the only usage? can we remove `github.com/go-kit/kit`?

Reply:
Let's have a separated PR to for that cleanup?

Reply:
https://github.com/app-sre/aws-resource-exporter/pull/144